system/scripts/zone/ais/basic_monster.cs

```diff
@@ -50,24 +52,29 @@ protected IEnumerable Idle()
 	protected IEnumerable Attack()
 	{
 		SetRunning(true);
-
```

Comment:
Just for my understanding, why are you removing all sensible spacing and grouping in this method?^^